website/community/committers/release.md

```diff
@@ -145,6 +146,10 @@ Please note that this version is the exact version of the release, not the relea
     - node.js: `bindings/nodejs/upgrade.md`
     - python: `bindings/python/upgrade.md`
 
+### Generate dependencies list
```

Comment:
Since we check deps updated in CI, we don't need this section anymore.

Reply:
These files have been submitted to the repo. We check them in ci but not update them. This makes the update need to be manually executed, then we still need this section.

Reply:
> We check them in ci but not update them

No. If CI failed, we should update them to pass. So the files must be up-to-date. Otherwise, CI workflow has a bug,

Reply:
> No. If CI failed, we should update them to pass. So the files must be up-to-date. Otherwise, CI workflow has a bug,

My current implementation of the CI workflow is optimized for contributors. We will check the dependences by `cargo deny check license` but not enfore the dependences list to be updated. (website is different for the limitation of tools).

In this way:

- Contributor don't need to take the job of generate dependences list (they don't need to install cargo-deny locally).
- Release Manager don't need to perform checks (since they have been checked for every CI running). They only need to generate dependences list.

Reply:
OK.